skunkworks_crow/src/main/java/org/odk/share/activities/ScannerActivity.java

```diff
@@ -0,0 +1,133 @@
+package org.odk.share.activities;
+
+/*
+ * created by Chromicle
+ */
+
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+
+import com.journeyapps.barcodescanner.CaptureManager;
+import com.journeyapps.barcodescanner.DecoratedBarcodeView;
+
+import org.odk.share.R;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+
+public class ScannerActivity extends AppCompatActivity implements
+        DecoratedBarcodeView.TorchListener {
+
+    private CaptureManager capture;
+    private DecoratedBarcodeView decoratedScannerView;
+    private Button switchFlashlightButton;
+    private boolean isFlashLightOn = false;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_scanner);
+        getWindow().addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+
+        //Initialize barcode scanner view
+
```

Comment:
Please remove the extra blank lines and the comment. It doesn't add much value to the code